AzureDevOpsTemplates/Build/StepTemplates/dfc-dotnetframework-build.yml

```diff
@@ -41,15 +46,9 @@ steps:
      codeCoverageEnabled: true
      platform: ${{ parameters.BuildPlatform }}
      configuration: ${{ parameters.BuildConfiguration }}
- - task: CopyFiles@2
-   displayName: 'Copy Files to: $(build.artifactstagingdirectory)/Azure'
-   inputs:
-     sourceFolder: 'Resources'
-     contents: '**'
-     targetFolder: '$(build.artifactstagingdirectory)/Azure'
- - task: PublishBuildArtifacts@1
+
+ - task: PublishPipelineArtifact@1
    displayName: 'Publish Artifact: ${{ parameters.SolutionBaseName }}'
    inputs:
-     pathtoPublish: $(build.artifactstagingdirectory)
-     artifactName: ${{ parameters.SolutionBaseName }}
-     publishLocation: container
\ No newline at end of file
+     targetPath: $(build.artifactstagingdirectory)
+     artifactName: ${{ parameters.SolutionBaseName }}
```

Comment:
Can this be updated to follow the same naming convention as the others?  ie suffixed with .DeploymentPackages